src/command/output.rs

```diff
@@ -24,26 +23,12 @@ impl RoverStdout {
     pub fn print(&self) {
         match self {
             RoverStdout::SDL(sdl) => {
-                // we check to see if stdout is redirected
-                // if it is, we don't print the content descriptor
-                // this is because it would look strange to see
-                // SDL:
-                // and nothing after the colon if you piped the output
-                // to another process or a file.
-                if atty::is(Stream::Stdout) {
-                    tracing::info!("SDL:");
```

Comment:
Oh, and I guess if we are removing this from `output.rs`, we out to make sure this is printed or traced at some other point. Otherwise (especially I think with `Schema Hash:`, it might be a bit confusing.